std_msgs/Header header
common/Uint8TimeStamped cdcu_brkcmdperd_err
common/Uint8TimeStamped cdcu_brkcmddrop_err
common/Uint8TimeStamped cdcu_brkcmdchksum_err
common/Uint8TimeStamped cdcu_brkcmdoffline_err
common/Uint8TimeStamped cdcu_brkcmdoverrge_err
common/Uint8TimeStamped cdcu_ehbmsgperd_err
common/Uint8TimeStamped cdcu_ehbmsgdrop_err
common/Uint8TimeStamped cdcu_ehbmsgchksum_err
common/Uint8TimeStamped cdcu_ehbmsgoffline_err
common/Uint8TimeStamped cdcu_brkfldlack_err
common/Uint8TimeStamped cdcu_ehbcurtsenor_err
common/Uint8TimeStamped cdcu_ehbtempsenor_err
common/Uint8TimeStamped cdcu_ehbovertemp_err
common/Uint8TimeStamped cdcu_brkpwrsupply_err
common/Uint8TimeStamped cdcu_brkpresrsenor_err
common/Uint8TimeStamped cdcu_ehbpwrsupply_err
common/Uint8TimeStamped cdcu_ecupwrsupply_err
common/Uint8TimeStamped cdcu_drvbridge_err
common/Uint8TimeStamped cdcu_drvmtr_err
common/Uint8TimeStamped cdcu_ehbcan_err